packages/odata-common/src/de-serializers/default-de-serializers.ts

```diff
@@ -0,0 +1,93 @@
+import { identity } from '@sap-cloud-sdk/util';
+import {
+  serializeFromBigNumber,
+  deserializeToNumber,
+  serializeFromNumber,
+  deserializeToBigNumber,
+  validateGuid,
+  convertToNumber
+} from './payload-value-converter';
+import { DeSerializationMiddleware } from './de-serialization-middleware';
+import { convertToUriForEdmString, isInfOrNan } from './uri-value-converter';
+
+export const defaultDeSerializersRaw: DeSerializationMiddleware = {
```

Comment:
What is the `raw` reffering to? That it is raw data? If so I would a a doc explaining what is the default and leave it out.

Reply:
I guess the `raw` is used with e.g., `wrapSerialization`, where:
- `raw` is the serialiser it self
- `wrap**` handles the `null` check.

Reply:
I am not happy with the naming either. Will try to improve before the next PR.